tools/multimodal/source_format_to_data_juicer_format/wavcaps_to_dj.py

```diff
@@ -0,0 +1,236 @@
+# This tool is used to convert multimodal dataset in WavCaps format to a target
+# dataset in Data-Juicer format.
+#
+# WavCps format:
+# { 'num_captions_per_audio': 1,
+#   'data': [{
+#       'title': 'Airplane Landing Airport',
+#       'description': 'Large commercial airplane landing at an airport runway.',  # noqa: E501
+#       'author': 'Daniel Simion',
+#       'href': '2219-Airplane-Landing-Airport.html',
+#       'caption': 'An airplane is landing.',
+#       'id': '2219',
+#       'duration': 14.1424375,
+#       'audio': 'wav_path',
+#       'download_link': 'http://soundbible.com/grab.php?id=2219&type=wav'
+#   },  {
+#       'title': 'Service Bell Help',
+#       'description': 'Customer ringing service bell in need of help in a store.',  # noqa: E501
+#       'author': 'Daniel Simion',
+#       'href': '2218-Service-Bell-Help.html',
+#       'caption': 'Someone is ringing a bell.',
+#       'id': '2218',
+#       'duration': 1.5698125,
+#       'audio': 'wav_path',
+#       'download_link': 'http://soundbible.com/grab.php?id=2218&type=wav'
+#   },
+#   ...]
+# }
+#
+# Corresponding Data-Juicer format:
+# {'audios': ['./path/to/audio/2219.flac'],
+#  'text': '<audio>\n'
+#          '[[caption]]: An airplane is landing. <|__dj__eoc|>',
+#  '__dj__meta__': {
+#       'num_captions_per_audio': 1,
+#       'title': 'Airplane Landing Airport',
+#       'description': 'Large commercial airplane landing at an airport runway.',  # noqa: E501
+#       'author': 'Daniel Simion',
+#       'href': '2219-Airplane-Landing-Airport.html',
+#       'caption': 'An airplane is landing.',
+#       'id': '2219',
+#       'duration': 14.1424375,
+#       'audio': 'wav_path',
+#       'download_link': 'http://soundbible.com/grab.php?id=2219&type=wav',
+#       'category': '',
+#       'tags': '' }}
+# {'audios': ['./path/to/audio/2218.flac'],
+#  'text': '<audio>\n'
+#          '[[caption]]: Someone is ringing a bell. <|__dj__eoc|>',
```

Comment:
The string "[[caption]]: " might be unnecessary? LLaVA format needs this "role" part to represent single/multi-turn conversations, but there is no need to add this part for WavCaps, which is actually a captioning dataset.